requirements.txt

```diff
@@ -41,3 +41,4 @@ transformers>=4.41.2 ; python_version >= '3.11'
 trl>=0.9.4
 wandb>=0.16.4
 xdg-base-dirs>=6.0.1
+psutil==6.0.0
```

Comment:
Any particular reason this isn't `psutil>=6.0.0`?

Reply:
switched to this